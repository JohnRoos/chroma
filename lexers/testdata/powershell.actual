Get-ChildItem -Recurse -Force -ErrorAction SilentlyContinue -Name -Path C:\ *.txt
$x = "Here is a $var inside a string"
$x = @"
Here is a $var inside a here-string
"@