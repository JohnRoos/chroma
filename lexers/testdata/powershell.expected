[
  {"type":"NameBuiltin","value":"Get-ChildItem"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"-Recurse"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"-Force"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"-ErrorAction"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"SilentlyContinue"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"-Name"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"-Path"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"C:"},
  {"type":"Punctuation","value":"\\"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"*."},
  {"type":"Name","value":"txt"},
  {"type":"Text","value":"\n"},
  {"type":"NameVariable","value":"$x"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"Here is a "},
  {"type":"NameVariable","value":"$var"},
  {"type":"LiteralStringDouble","value":" inside a string\""},
  {"type":"Text","value":"\n"},
  {"type":"NameVariable","value":"$x"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"Text","value":" "},
  {"type":"LiteralStringHeredoc","value":"@\"\nHere is a "},
  {"type":"NameVariable","value":"$var"},
  {"type":"LiteralStringHeredoc","value":" inside a here-string\n\"@"}
]
